src/main/java/com/carmen/app/control/PersistenceService.java

```diff
@@ -0,0 +1,45 @@
+package com.carmen.app.control;
+
+import java.util.List;
+
+import javax.ejb.Stateless;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.TypedQuery;
+
+
+import com.carmen.app.utils.Logged;
+
+@Stateless
+@Logged
+public class PersistenceService<T,K>{
+
+	
+	@PersistenceContext(unitName = "carP")
+	private EntityManager em;
+	
+	public List<T> getEntities(String NameQuery, Class<T> c){
```

Comment:
@carmopi instead of using the word **"Entity"** in the methods name, it would be much better if you omit it. I mean, the user of this class knows it is dealing with entities 😄 

So, I would suggest names like: getAll or getById or delete.. things like that.